docs/1.0. Certificate Provisioning.md

```diff
@@ -68,46 +68,48 @@ The entity that is using the NMOS API, for example:
 ## Introduction (informative)
 
 This document covers the automated provisioning of TLS Certificates to NMOS Servers, which are then used to secure communications between NMOS Servers and NMOS Clients.
+What about provisioning of the network's Root CA to NMOS Clients?
 
 This document is not concerned with the security of the connection used to carry out provisioning of the TLS Certificate, but for the mechanism described in this document to be effective the connection must be secured, ideally using the recommendations covered in [BCP-003-01][].
 
 ## Automated Certificate Provisioning Flow (informative)
```

Comment:
Feels like an informative section shouldn't use normative language (should, must, etc.)

This section is also duplicated in part lower down...

Reply:
> This document covers the automated provisioning of TLS Certificates to NMOS Servers, which are then used to secure communications between NMOS Servers and NMOS Clients.

This document covers the automated provisioning of TLS Certificates to NMOS Servers and automated provisioning of CA Certificate to NMOS Clients, which are then used to secure communications between them.

- Create 2 separate `Automated Certificate Provisioning Flow (informative)` sessions, one for NMOS Client and one for NMOS Server.
- Replace `Nodes NMOS APIs` with `NMOS Server's APIs` in the `Automated Certificate Provisioning Flow `diagram.
- Should NMOS Client re-get Root CA in CA half-life?

Reply:
Sounds like a good idea to me, @simonlo-sony.